tests/pytest/oauth/test_views.py

```diff
@@ -71,6 +73,40 @@ def test_authorize_success(mocked_oauth_client_instance, mocked_analytics_module
     assert result.url == reverse(ROUTE_CONFIRM)
 
 
+@pytest.mark.django_db
+def test_authorize_success_with_claim(mocked_session_verifier_auth_required, mocked_oauth_client_instance, app_request):
+    # mocked_session_verifier_auth_required is a fixture that mocks benefits.core.session.verifier(request)
+    # call it here, passing a None request, to get the return value from the mock
+    verifier = mocked_session_verifier_auth_required(None)
```

Comment:
Not a blocker, but the other syntax I used on that new fixture could be used here too (I might go back and clean all these up, this seems more clear):

```python
verifier = mocked_session_verifier_auth_required.return_value
```